docs/index.md

```diff
@@ -0,0 +1,562 @@
+---
+layout: page-fullwidth
+order: 1
+title: "Hand Coded Heat"
+subheadline: "Hello World for Numerical Packages"
+teaser: "Why use numerical packages..."
+permalink: "lessons/hand_coded_heat/"
+use_math: true
+lesson: true
+header:
+ image_fullwidth: "Differential-Equations-e1509686869201.png"
+---
+
+## At a Glance
+
+|Questions|Objectives|Key Points|
+|:--------|:---------|:---------|
+|What is a numerical algorithm?|Implement a simple numerical algorithm.<br>Use it to solve a simple science problem.|Numerical Packages are used<br>to solve scientific problems involving [PDEs.][PDE]|
+|What is [discretization][DISC]?|Introduce basic concepts in solving continuous<br>[PDEs][PDE] using discrete computations.|_Meshing_ (or [discretization][DISC]) is an<br>important first step.|
+|How can numerical packages<br>help me with my software?|Understand the value numerical packages<br>offer in developing science applications|Numerical packages offer: rigorous/vetted numerics<br>greater generality, extreme scalability and more...|
+
+#### To begin this lesson
+
+* Go to the directory for the hand-coded `heat` application
+```
+cd {{ site.handson_root }}/hand_coded_heat
+```
+
+## A Simple Science Question
+
+Will my pipes freeze during a long, cold storm?
+
+![Wall and Pipe::](assets/wall_and_pipe.png){:width="500px"}
```

Comment:
All cases where you have `assets/<image file name>` need to be changed to just `<image-file-name>` because you are keeping the images all at the same level as the `.md` file you are referencing them from.

Or, you can add a `assets` directory if you like and then move all the images to that directory. I prefer not doing that just because its more complexity without much more value. But, you choose.